src/index.js

```diff
@@ -11,6 +11,11 @@ injectGlobal`
   * {
     box-sizing: border-box;
   }
+
+  body, html {
+    font-size: 16px;
+    font-family: 'Roboto', sans-serif;
```

Comment:
Ugh, wasn't there an uglier typeface to use? Nvm, I'll change the text to Lato and the code to Inconsolata.